File: ezSCORES/ezSCORES.Model/Goals.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class Goals
	{
		public int Id { get; set; }

		public int? CompetitionTeamPlayerId { get; set; }

		public int MatchId { get; set; }

		public int SequenceNumber { get; set; }

		public int ScoredAtMinute { get; set; }

		public bool IsHomeGoal { get; set; }
	}
}

File: ezSCORES/UI/ezscores_desktop/lib/models/goals.dart
import 'package:json_annotation/json_annotation.dart';
import 'competitionsTeamsPlayers.dart';

part 'goals.g.dart';

@JsonSerializable()
class Goals {
	int? id;
	int? competitionTeamPlayerId;
	int? matchId;
	int? sequenceNumber;
	int? scoredAtMinute;
	bool? isHomeGoal;
	CompetitionsTeamsPlayers? competitionsTeamsPlayers;

	Goals();

	factory Goals.fromJson(Map<String, dynamic> json) => _$GoalsFromJson(json);

	Map<String, dynamic> toJson() => _$GoalsToJson(this);
}